db/schema.rb

```diff
@@ -98,20 +98,20 @@
   add_index "profiles", ["user_id"], name: "index_profiles_on_user_id"
 
   create_table "requests", force: :cascade do |t|
-    t.datetime "created_at",                         null: false
-    t.datetime "updated_at",                         null: false
+    t.datetime "created_at",             null: false
+    t.datetime "updated_at",             null: false
     t.integer  "form_of_address"
     t.string   "first_name"
     t.string   "last_name"
     t.string   "phone_number"
-    t.string   "address"
+    t.string   "street"
     t.string   "email"
     t.text     "topic_of_workshop"
     t.text     "time_period"
     t.integer  "number_of_participants"
     t.string   "knowledge_level"
     t.text     "annotations"
-    t.integer  "status",                 default: 0
```

Comment:
Race condition with #330. This line was wrongly committed earlier but will be needed soon. If you merge before #330 gets merged, this would be appropriate, though a notice might be great. If this gets merged afterwards, you'd have to remove this change, e.g. leave this line in here.